### PR TITLE
Comments prevent loop

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?

There is a [discussion API bug](https://github.com/guardian/discussion-api/issues/395) that means we do not return the correct results when requesting some comments - which meant there was a loop that when checking for a particular comment, if it didn't exist, we would send the same request.

This prevents that loops, shows an error message to the user and pings Sentry.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/discussion/loader.js [hunk 1]
```diff
@@ -421,9 +421,25 @@ Loader.prototype.gotoComment = function(id) {
...
+        scroller.scrollToElement(qwery('.js-discussion-toolbar'), 100);
+        fastdom.write(function(){
+            $('.js-discussion-main-comments').prepend('<div class="d-discussion__message d-discussion__message--error">The comment you requested could not be found.</div>');
+        });
+
+        // Capture in sentry
+        raven.captureMessage('Comment doesn\'t exist in response', {
+            level: 'error',
+            extra: {
+                commentId: id
```
Comment (gidsg) [inline static/src/javascripts/projects/common/modules/discussion/loader.js]: Wonder if there's any more context we can include here like sort options, or the URL requested in NextGen API?

**Comment by gidsg** [comment]
LGTM 🚨 👍

File: static/src/javascripts/projects/common/modules/discussion/loader.js (hunk 1 above)
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/discussion/loader.js]: I agree, I am going to merge this so we are no longer loopdalooping, then I will add that and I also want to add that same info to a contact us link in context. Cheers.

**Comment by nicl** [comment]
Are we sure that the error can't occur when going via top comments (aka highlights/staff picks)? I.e. at the moment the logic is:

    goto comment
      if found -> show + focus
      else if !fromRequest (i.e. permalink) -> loop
      else -> error message

Instead, we could do:

    goto comment
      if found -> ...
      else -> attempt load and if fails error

I can't think why the issue/error should only affect permalinks and not staff picks?

**Pull request merged**